script/upgrades/MU03/MU03Checks.sol

```diff
@@ -102,6 +106,7 @@ contract MU03Checks is Script, Test {
     setUp();
 
     verifyOwner();
+    verifyEUROCSetUp();
```

Comment:
verify that pricing modules were added to BiPoolManager

Reply:
I could do that but with the latest changes to the BiPoolManager the createExchange function enforces this itself, so not sure if the explicit check is needed.

Reply:
ok let's leave it then 👍🏼